### DISCUSSION TITLE
20241003 Weekly Development Meeting

### DISCUSSION THREAD

**Discussion by josecelano** [nautilus-cyberneering/.github#110] [Meetings]
- Time:  9:30 in Canary Islands (WET (UTC+0)).
- Time: 10:30 in Vaduz (CET (UTC+1)).

Location:
<https://meeting.blockfinance-eco.li/NautilusCyberneeringWeeklyDevMeeting>

Expected duration: 2 to 4 hours with breaks

Communicated attendants:

- Cameron Garnham(@da2ce7)  - x 
- Jose Celano(@josecelano) - x
- Constantin Bosse(@cgbosse) - x
- Mario Melchor(@mario-nt) - x
- Graeme Byrne(@grmbyrn) - x

-------------------------------------------------------------

## Agenda:

1. Everyone will take their turn and:
- Go over his notes added as comments for the meeting which will have been added in advance to give an update of all the work done during the previous week indicating.
	- The issues worked on
	- The progress made on them
	- Other things done
- Inform of any faced challenges and request help if necessary to work on specific issues.
- Highlight the encountered difficulties, especially if there is a risk of them causing delays or affecting end product's quality.
- Communicate what will be the next week's tasks.
- List any new topics for further discussion in an additional Technical meeting that he noted for future discussion in comments to this discussion either before the meeting or after his turn so that they get considered.

During the meeting every team member will take their turns and follow this sequence.
The topics for further discussion will be noted in advance by each the team member and to the meetings agenda.

2. At the end of the meeting if there any new technical topics to be discussed further it will be decided to
set another meeting date for the discussion of these or to hold the meeting hereafter.

Also the persons to be part of the meeting will be decided on so that the others will be free to continue their own work.

**Comment by josecelano** [comment]
### Torrust

Released version 3.0.0

- https://github.com/torrust/torrust-tracker/releases/tag/v3.0.0
- https://github.com/torrust/torrust-index/releases/tag/v3.0.0
- https://github.com/torrust/torrust-index-gui/releases/tag/v3.0.0
  - https://github.com/torrust/torrust-index-api-lib/releases/tag/v3.0.0
  - https://github.com/torrust/torrust-index-types-lib/releases/tag/v3.0.0

### Torrust Website 2.0

- Reviewed and improved all contents
- Published new version: https://torrust.com/

PR reviews:

- https://github.com/torrust/torrust-website/pull/50

### Torrust Tracker 

Issues:

- Fixed https://github.com/torrust/torrust-tracker/issues/1052

### Torrust index 

Issues:

- Fixed https://github.com/torrust/torrust-index/issues/738

### Events

Attended this workshop: [Workshop: Learn to design architectures](https://www.meetup.com/gofigeeks/events/303466572/?eventOrigin=group_past_events)

I am organizing a Rust meetup on the 24th of October with @JesusPerez.

**Comment by mario-nt** [comment]
### GUI

- https://github.com/torrust/torrust-index-gui/pull/625

### Index
	
- https://github.com/torrust/torrust-index/pull/742


### Training and research

- Flexbox 101
- Research on how to structure releases and branches for the project

**Comment by grmbyrn** [comment]
## Torrust website v2
* [List of issues worked on in this PR](https://github.com/torrust/torrust-website/pull/50)
* [List of issues worked on in this PR](https://github.com/torrust/torrust-website/pull/68)
